### ISSUE TITLE
[Bug Report][3.3.21] Accessibility: Nested interactive components trigger accessibility warnings in v-select

### ISSUE THREAD

**Issue by kmmerc18** [vuetifyjs/vuetify#18495]
### Environment
**Vuetify Version:** 3.3.21
**Vue Version:** 3.3.4
**Browsers:** Chrome 117.0.0.0
**OS:** Mac OS 10.15.7

### Steps to reproduce
Use the aXe Accessibility tool (https://chrome.google.com/webstore/detail/axe-devtools-web-accessib/lhdoppojpmngadmnindnejefpokejbdd) to scan the page for WCAG 2.1 issues -> there should be one about "Interactive controls must not be nested"

### Expected Behavior
There should be no nested interactive controls, especially since v-select is a single component

### Actual Behavior
The automatic issue is being triggered
<img width="427" alt="Screenshot 2023-10-18 at 9 57 34 AM" src="https://github.com/vuetifyjs/vuetify/assets/60188830/d7774a84-4b09-4d23-8c9e-84decf577eb6">

### Reproduction Link
[https://play.vuetifyjs.com/#...](https://play.vuetifyjs.com/#eNpVj8EOgjAQRH+l2QsXhCieCJIYEv0ATTwAhworaVLaplSiMf67pRWJp519u5nMlC8YdBPvlYrGO0IKmcFecWowrwQh2biiSjnplkYKQ5lA/UUODsixMTMghNMr8l0FJ89huaTMug/2VAYF5ewmtWA0CElQSC41beWkD1ay1uEjSt35jzM+6DCJy1P2THRBvRjnWTynmKNa8J91Ir5KFv8qwjuEJEqiLfi5WUP9AVruUos=)
